DataHub/Integration/02_ManageAttributeSets.md

```diff
@@ -229,9 +225,9 @@ All required attributes are marked with a *(required)* indication in the product
 Note that entities can be saved even if not all *(required)* attributes are completed.   
 
 You can define whether an attribute should be required or not.
-Note that inherited attributes can only be changed in the origin attribute set and the required status is also inherited to the linked attribute sets.   
+Bear in mind that inherited attributes can only be changed in the origin attribute set and the required status is also inherited to the linked attribute sets.   
 
-Be aware that the activation or deactivation of languages affects the completeness calculation.
+Be aware that the activation or deactivation of languages and scopes affects the completeness calculation.
```

Comment:
...completeness calculation in the PIM module?

Reply:
Hmmm, guter Punkt. Die Frage ist, ob "entity" completeness nur relevant für PIM (und evtl. Omni-Channel) ist... Ich hinterlasse ein Kommentar für den Fachreviewer.